es6-destructuring/index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>ES6 Destructuring</title>
+</head>
+<body>
+  <script>
+    const book1 = {
+      title: 'Goodnight Punpun',
+      author: 'Inio Asano',
+      libraryID: 3353
+    };
+
+    const {
+      title: book1Title,
+      author: book1Author,
+      libraryID: book1LibraryID
+    } = book1;
```

Comment:
The first three destructured constants should have the same name as `book1`'s property names.

---

![image](https://user-images.githubusercontent.com/16143771/174709012-f25353e0-fd56-4b2c-ad8d-2f76b5abeb6c.png)